extra/update-wiki-version.js

```diff
@@ -39,6 +43,10 @@ function updateWiki(newVersion) {
     safeDelete(wikiDir);
 }
 
+/**
+ * Check if a directory exists before deleting
```

Comment:
IMO, it's not only "checking" :)

```suggestion
 * Check and delete existing directory
```